pynicotine/gtkgui/frame.py

```diff
@@ -905,15 +906,22 @@ def on_connect(self, *args):
             self.np.servertimer.cancel()
             self.np.servertimer = None
 
+        self.away = self.np.config.sections["server"]["away"]
```

Comment:
It would be better to set this at https://github.com/Nicotine-Plus/nicotine-plus/blob/master/pynicotine/pynicotine.py#L1050, since the UI is initialized with the away status at that point. A SetStatus message is also sent there.

Reply:
I agree!

But I think it's still no good to put it in login(), because it would make the "chosen away state" to be set at every login, conflicting with the real away state.

I moved it to NicotineFrame initialization.